packages/block-library/src/button/edit.native.js

```diff
@@ -310,13 +282,6 @@ class ButtonEdit extends Component {
 				? borderRadiusValue + spacing + borderWidth
 				: 0;
 
-		// To achieve proper expanding and shrinking `RichText` on iOS, there is a need to set a `minWidth`
-		// value at least on 1 when `RichText` is focused or when is not focused, but `RichText` value is
-		// different than empty string.
-		const minWidth =
-			isButtonFocused || ( ! isButtonFocused && text && text !== '' )
-				? MIN_WIDTH
-				: placeholderTextWidth;
```

Comment:
This is not needed anymore for iOS and it's working in the same way as it was before, but on Android this change seems to break the placeholder minimum width.

@jd-alexander - we need to test the `Buttons` block also while working on this.

Reply:
@etoledom just to verify could we keep the previous implementation for Android? I did some checks and so far I haven't found a good way to replicate the behavior that's done in the iOS native code as there's no way to easily measure the placeholder text and set that as the width of the text component. I am going to have another look tomorrow to see if anything comes up.

Reply:
We could, but the previous behavior on Android still looks like the one in the first screenshot (With "Downloa`\n`d").
If we have a solution only for that one and not for placeholder, I guess we can revert the placeholder changes for Android.

Reply:
Okay, yes, I am working on a solution involving this which should be measuring the placeholder and using that value as the minimum witdth. Just to clarify this is how the issues looks on Android.

Placeholder | Text 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/100174164-e31af200-2e99-11eb-993d-d5e1656dd1e3.png" width="320"></kbd>     |     <kbd><img src="https://user-images.githubusercontent.com/1509205/100174171-e57d4c00-2e99-11eb-8005-43943c9dc738.png" width="320"></kbd>